.github/ISSUE_TEMPLATE/-maintainer--new-feature.md

```diff
@@ -0,0 +1,20 @@
+---
+name: "(Maintainer) New Feature"
+about: New feature description, filed by CRM maintainers
+title: ''
+labels: enhancement, source::maintainer
+assignees: ''
+
+---
+
+**What is the idea?**
+Describe what the feature is and the desired state.
+
+**Why is this needed?**
+Who would benefit from this feature? Why would this add value to them? What problem does this solve?
```

Comment:
I like the customer-focused approach here, but this feels like more of a design document question. See my next comment below.